client/my-sites/site-settings/form-jetpack-monitor.jsx

```diff
@@ -1,212 +1,179 @@
-					<p>
-						{ this.translate(
-							"We'll periodically check your site from our global network of servers to make sure it's online, and email you if it looks like your site is not responding for any reason."
-						) }
-					</p>
-				</div>
+	saveSettings = () => {
+		const { siteId } = this.props;
 
-			</div>
-		);
-	},
+		this.props.updateSiteMonitorSettings( siteId, this.state );
+	}
 
-	saveSettings: function() {
-		this.setState( { submitingForm: true } );
-		notices.clearNotices( 'notices' );
-		this.props.site.updateMonitorSettings( this.state.emailNotifications, this.state.wpNoteNotifications, ( function( error ) {
-			if ( error ) {
-				this.handleError();
-				notices.error( this.translate( 'There was a problem saving your changes. Please, try again.' ) );
-				return;
-			}
-			notices.success( this.translate( 'Settings saved successfully!' ) );
-			this.props.markSaved();
-			this.setState( { submittingForm: false } );
-		} ).bind( this ) );
-	},
+	settingsMonitorEmailCheckbox() {
+		const { monitorActive, translate } = this.props;
 
-	settings: function() {
 		return (
-			<div>
-				<p>{ this.translate( "Jetpack is currently monitoring your site's uptime." ) }</p>
-				{ this.settingsMonitorEmailCheckbox() }
-				{ config.isEnabled( 'settings/security/monitor/wp-note' ) ? this.settingsMonitorWpNoteCheckbox() : '' }
-			</div>
+			<CompactFormToggle
+				disabled={ this.disableForm() || ! monitorActive }
+				onChange={ this.handleToggle( 'email_notifications' ) }
+				checked={ !! this.state.email_notifications }
+			>
+				{ translate( 'Send notifications to your {{a}}WordPress.com email address{{/a}}.', {
+					components: {
+						a: (
+							<a
+								href="/me/account"
+								onClick={ this.recordEvent( 'Clicked on Monitor Link to WordPress.com Email Address' ) }
+							/>
+						)
+					}
+				} ) }
+			</CompactFormToggle>
 		);
-	},
+	}
 
-	settingsMonitorEmailCheckbox: function() {
+	settingsMonitorWpNoteCheckbox() {
+		const { monitorActive, translate } = this.props;
 		return (
-			<FormLabel>
-				<FormCheckbox
-					disabled={ this.disableForm() }
-					onClick={ this.recordEvent.bind( this, 'Clicked on Monitor email checkbox' ) }
-					id="jetpack_monitor_email"
-					checkedLink={ this.linkState( 'emailNotifications' ) }
-					name="jetpack_monitor_email" />
-				<span>
-					{ this.translate( 'Send notifications to your {{a}}WordPress.com email address{{/a}}.', {
-						components: {
-							a: <a href="/me/account" onClick={ this.recordEvent.bind( this, 'Clicked on Monitor Link to WordPress.com Email Address' ) } />
-						}
-					} ) }
-				</span>
-			</FormLabel>
+			<CompactFormToggle
+				disabled={ this.disableForm() || ! monitorActive }
+				onChange={ this.handleToggle( 'wp_note_notifications' ) }
+				checked={ !! this.state.wp_note_notifications }
+			>
+				{ translate( 'Send notifications via WordPress.com notification.' ) }
```

Comment:
:+1: indeed; addressed in https://github.com/Automattic/wp-calypso/pull/13281/commits/d55005f4fe8802a8d1de2c131a5d8764a1faa3e6.